lib/crypto_importers/mappers/coinmetro_mapper.rb

```diff
@@ -0,0 +1,25 @@
+class CoinmetroMapper < BaseMapper
+  tag Tag::COIN_METRO
+  mappings [
+    {
+      id: 'coinmetro-transactions',
+      required_headers: %w[Currency Date Description Amount Fees Price Pair Other\ Currency Other\ Amount],
+      optional_headers: %w[IBAN Transaction\ Hash Additional\ Info],
+      header_mappings: {
+        date: 'Date',
+        from_amount: 'Amount',
+        from_currency: 'Currency',
+        to_amount: 'Other Amount',
+        to_currency: 'Other Currency',
```

Comment:
The `from_amount` and `from_currency` are set incorrectly

```suggestion
        amount: 'Amount',
        currency: 'Currency',
```

It should be:
`currency: 'Currency',`
`amount: 'Amount',`

`Other amount` and `Other Currency` are simply the same amount in fiat currency

In dollars, for example

This is not `from_amount` and not `from_currency`

Reply:
Updated